### PR TITLE
doc(k8s) re-do and update the Kong w\ k8s installation docs

### PR DESCRIPTION
**Pull request by hutchic**
- mention the existence of the Make tasks in kong-dist-kubernetes
- change k8s cluster to a prerequisite don't document how to provision one
- similarly let's not document how to use minikube
- demonstrate how to access the admin-api / proxy and continue the getting started tutorial at adding a configuring a service

To do but out of scope is we need to duplicate all relevant admin api examples for declarative. Right now it's not clear where best to direct folks after they finish setting up declarative / db-less Kong

To do in the a PR to follow this one is continue the k8s w\ datastore examples to include setting up https://github.com/Kong/kubernetes-sidecar-injector and running bookinfo.yaml

### CODE REVIEW

File: app/install/kubernetes.md [hunk 1]
```diff
@@ -56,145 +54,141 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
 
 <iframe width="660" height="400" src="https://www.youtube-nocookie.com/embed/MPlSyWDAOpw?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-# Kong via Helm or Minikube
+# Kong via Helm
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
## Kong via Helm
```

File: app/install/kubernetes.md [hunk 2]
```diff
@@ -56,145 +54,141 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
 
-An easy way to deploy Kong on Kubernetes is via [Helm]({{ page.links.kubeapps_hub }}).
+Install Kong or Kong Enterprise using the official [Helm chart]({{ page.links.kubeapps_hub }})
 
-Kong can also be deployed on minikube - please follow the [README]({{ page.links.minikube }})
-and use the manifest files provided in `minikube` directory.
+For questions and discussion, please visit [Kong Nation](https://discuss.konghq.com/c/kubernetes).
 
 # Kong via Manifest Files
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
## Kong via Manifest Files
```

**Comment by levfishbluefish** [comment]
Added [PR #1357](https://github.com/Kong/docs.konghq.com/pull/1357) -> suggest merging that PR first, then you can update this page to use the proper heading tags and use the generated TOC

File: app/install/kubernetes.md [hunk 3]
```diff
@@ -56,145 +54,141 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
 
 <iframe width="660" height="400" src="https://www.youtube-nocookie.com/embed/MPlSyWDAOpw?rel=0" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
 
-# Kong via Helm or Minikube
+# Kong via Helm
 
-An easy way to deploy Kong on Kubernetes is via [Helm]({{ page.links.kubeapps_hub }}).
+Install Kong or Kong Enterprise using the official [Helm chart]({{ page.links.kubeapps_hub }})
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
Install **Kong** or **Kong Enterprise** using the official [Helm chart]({{ page.links.kubeapps_hub }})
```

File: app/install/kubernetes.md [hunk 4]
```diff
@@ -30,7 +28,7 @@ Install Kong or Kong Enterprise using the official
 <a href="https://github.com/Kong/kubernetes-ingress-controller">Kubernetes Ingress Controller</a>.
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]: Please change all hrefs to markdown syntax

File: app/install/kubernetes.md [hunk 5]
```diff
@@ -56,145 +54,141 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]: Change lines 47 and 49 to markdown links

**Comment by levfishbluefish** [comment]
Added a few minor markdown fixes for links, once those are in and the heading tags are adjusted to use the toc generator, this PR is GTG. 

There is a larger issue of wanting to expand these docs vs keeping most of the information in github with this doc mainly be a reference to those, but that is out of scope of this PR and we should go ahead and merge it 👍

**Comment by hishamhm** [comment]
@levfishbluefish I had started reviewing this yesterday but was preempted halfway (mostway? ;) ) through — I won't have time to fully rewiew, so I just hit "Send" on the suggestions I had lying around, so these don't get lost before I go out on vacation. It's all style/readability stuff though, not a technical review on the content.

File: app/install/kubernetes.md [hunk 6]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
+
+Now deploy the Kong dataplane using the `kong-dbless.yaml` file from this repository
 
 ```bash
-$ kubectl delete -f kong_migration_<postgres|cassandra>.yaml
+$ kubectl apply -f kong-dbless.yaml
 ```
 
-### 5. **Deploy Kong**
+### **Using Declarative / DB Less Backed Kong**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
### Using Declarative / DB Less Backed Kong
```

File: app/install/kubernetes.md [hunk 7]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
 
-**Kong Enterprise trial users** should complete the steps in the
-**Additional Steps for Kong Enterprise Trial Users** section below before proceeding.
-
-### 4. **Prepare database**
+Continue to [configuring a service](/latest/getting-started/configuring-a-service/).
 
-Using the `kong_migration_<postgres|cassandra>.yaml` file,
-run the migration job:
+### **Using Kong without a Database**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
### Using Kong without a Database
```

File: app/install/kubernetes.md [hunk 8]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
+``` bash
+$ kubectl -n kong apply -f kong-ingress-data-plane-postgres.yaml
+```
 
-Before deploying Kong, you need to provision a Cassandra or PostgreSQL pod.
+Continue to [Using Datastore Backed Kong](#using-datastore-backed-kong)
 
-For Cassandra, use the `cassandra.yaml` file from this repo to deploy a
-Cassandra `Service` and a `StatefulSet` in the cluster.
+### **Using Datastore Backed Kong**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
### Using Datastore Backed Kong
```

File: app/install/kubernetes.md [hunk 9]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
-    ```
+``` bash
+$ kubectl -n kong apply -f kong-ingress-data-plane-cassandra.yaml
+```
 
-2. **Install kubectl**
+Continue to [Using Datastore Backed Kong](#using-datastore-backed-kong)
 
-    If you already have [kubectl]({{ page.links.kubectl }}) installed, you can skip this step.
+### **PostgreSQL Backed Kong**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
### PostgreSQL Backed Kong
```

File: app/install/kubernetes.md [hunk 10]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
 
-Using the `cluster.yaml` file from this repo, deploy a
-GKE cluster. Provide the following information before deploying:
+For DB-less mode, continue to [Using Kong without a Database](#using-kong-without-a-database)
 
-1. Desired cluster name
-2. Zone in which to run the cluster
-3. A basicauth username and password for authenticating the access to the
-cluster
+### **Cassandra Backed Kong**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
### Cassandra Backed Kong
```

File: app/install/kubernetes.md [hunk 11]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
 
 ## Kong via Manifest Files
 
 Kong, or the trial version of Kong Enterprise, can be provisioned
 on a Kubernetes cluster via the manifest files provided
-in the [repo]({{ page.links.gh_repo }}).
+in the [Kong Kubernetes repository]({{ page.links.gh_repo }}).
 
-### 1. **Initial setup**
+### **Prerequisites**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
### Prerequisites
```

File: app/install/kubernetes.md [hunk 12]
```diff
@@ -47,145 +44,149 @@ once you've completed your experiment to avoid on-going Google Cloud usage charg
...
 
-Download or clone the following repo:
+1. Download or clone the [Kong Kubernetes repository]({{ page.links.gh_repo }})
+2. A Kubernetes cluster
 
+## **Installation Steps**
```
Comment (levfishbluefish) [inline app/install/kubernetes.md]:
```suggestion
## Installation Steps
```

**Pull request merged**